front/public/locales/en/stdcm.json

```
@@ -7,6 +7,7 @@
   },
   "loaderImageLegend": "The TGV Nord line",
   "notificationTitle": "Phase 1: from D-7 to D-1 5pm, on the Perrigny-Miramas axis.",
+  "pathfindingFailed": "No paths have been found for these waypoints.",
```

To be synced with the french trad where we don't use plural

```suggestion
  "pathfindingFailed": "No path has been found for these waypoints.",
```